src/pages/Filing/UpdateFinancialProfile/types.ts

```diff
@@ -5,6 +5,50 @@ export interface CheckboxOption {
   label: string;
 }
 
+// TODO: Send checkboxes values as {'1': true }?
```

Comment:
For future work: I'd think we should probably send it as something human readable with just the label, since SBL Help will need to modify it themselves?